Campaign_Manager/models.py

```diff
@@ -1,29 +1,70 @@
 from django.db import models
 from Character_Builder.models import Character
+from django.contrib.auth.models import User
 
 # Constants
 MAX_LENGTH_CAMPAIGN_NAME = 255
 
+# Automatically creates CampaignIDs
+def newCampaignID():
+    previousCampaign = Campaign.objects.last()
+    if not previousCampaign is None:
+        newID = previousCampaign.campaignID + 1
+    else:
+        newID = 0
+
+    return newID
+
+# finds a default user
+def defaultUser():
+    default = User.objects.first()
+
+    if default is None:
+        default = User.objects.create_user('defaultUser', password='djangoproject')
+
+    return default
+
 # Keeps track of individual campaigns
 class Campaign(models.Model):
-    campaignID = models.IntegerField(unique=True)
+    creator = models.ForeignKey(User, on_delete=models.CASCADE, default=defaultUser)
+    campaignID = models.IntegerField(unique=True, default=newCampaignID, editable=False)
     campaignName = models.CharField(max_length = MAX_LENGTH_CAMPAIGN_NAME)
 
     def __str__(self):
         return self.campaignName
 
+# Automatically creates CampaignDMIDs
+def newCampaignDMID():
```

Comment:
I don't think these newID functions are necessary. You can set an auto-incrementing primary key with:

models.AutoField(primary_key=True)

See: https://docs.djangoproject.com/en/1.11/topics/db/models/#automatic-primary-key-fields